data/mods/More_Survival_Tools/recipes.json

```diff
@@ -1040,9 +1090,68 @@
     {"id":"HAMMER","level":1,"amount":1}
   ],
   "components": [
-    [ [ "rock", 5 ] ],
+    [ [ "rock", 5 ], [ "ceramic_shard", 1 ], [ "sharp_rock", 1 ] ],
```

Comment:
Why only 1 shard/sharp rock but 5 regular rocks?

Also, at 3 super/bone glue it is not worth the effort. 1 would be fine.